alienitems.xml

```diff
@@ -0,0 +1,2001 @@
+    <Sprite texture="Content/Items/Alien/AlienRuins_Items.png" depth="0.6" sourcerect="542,112,102,99" origin="0.5,0.5" />
+    <Deconstruct time="30">
+      <Item identifier="steel" />
+      <Item identifier="incendium" />
+      <Item identifier="incendium" />
+      <Item identifier="incendium" />
+    </Deconstruct>
+    <Body radius="40" density="11" friction="0.95" />
+    <Holdable slots="RightHand+LeftHand" holdpos="30,-15" handle1="0,10" handle2="0,-10" msg="ItemMsgPickUpSelect">
+      <StatusEffect type="NotInWater" target="This" stackable="false" delay="30" checkconditionalalways="true">
+        <Conditional hastag="neq artifactcontainer" targetcontainer="true" />
+        <Conditional voltage="lte 0.0" targetcontainer="true" />
+        <ParticleEmitter particle="thermalSetFire" anglemax="360" scalemultiplier="3,3" particleamount="40" />
+        <sound file="Content/Items/Alien/AlienTurret1.ogg" range="500.0" loop="false" />
+        <Fire />
+      </StatusEffect>
+      <StatusEffect type="Always" target="This">
+        <Conditional hastag="neq artifactcontainer" targetcontainer="true" />
+        <Conditional voltage="lte 0.0" targetcontainer="true" />
+        <ParticleEmitter particle="thermalfx" anglemax="360" distancemax="5" particlespersecond="40" />
+        <sound file="Content/Items/Alien/AlienArtifactLoop1.ogg" range="500.0" loop="true" />
+      </StatusEffect>
+    </Holdable>
+        <!-- Emits steam periodically when in water, burns. Had to use Lightcomponent to make it work -->
+    <LightComponent range="150.0" lightcolor="210,120,30,120" IsOn="False" castshadows="false" allowingameediting="false" pulseamount="0.1" pulsefrequency="4">
+      <Statuseffect type="InWater" target="This" IsOn="True" Interval="20" />
+      <Statuseffect type="NotInWater" target="This" IsOn="False" />
+      <StatusEffect type="OnActive" target="This" IsOn="False" delay="5" stackable="False" />
+      <StatusEffect type="OnActive" target="This" checkconditionalalways="true" >
+        <Conditional hastag="neq artifactcontainer" targetcontainer="true" />
+        <Conditional Voltage="lte 0.0" targetcontainer="true" />
+        <ParticleEmitter particle="bubbles" particlespersecond="5" anglemin="60" anglemax="120" scalemin="1" scalemax="2" velocitymin="5" velocitymax="100" copyentityangle="false" drawontop="true"/>
+        <ParticleEmitter particle="steamspray" particlespersecond="30" anglemin="60" anglemax="120" scalemin="1" scalemax="2" velocitymin="300" velocitymax="500" copyentityangle="false" highqualitycollisiondetection="true" drawontop="true" />
+        <sound file="Content/Items/Weapons/FlameThrowerLoop.ogg" type="OnUse" range="700.0" loop="true" />
+        <!-- burns via explosion -->
+        <Explosion range="200" structuredamage="0.1" itemdamage="0" smoke="False" flames="false" force="0" sparks="false" shockwave="False" underwaterbubble="false" camerashake="2" flash="false" playtinnitus="false" >

```

Comment:
It seems like executing this every frame might be bad for performance. Could probably be run once per second or so with no noticeable effect on the behavior.

Reply:
Unfortunately adding a 1 second interval breaks the sound and the particles, a 0.1 second interval keeps it "smooth". With a bigger interval and you see gaps between particles and it sounds like a steam engine

I have tried turning it into a repair tool that uses itself to no avail, it doesnt damage the holder

Reply:
Split the Statuseffect into two, assuming the explosion was the most performanceh hungry. Not sure if it's better.